_includes/default-header.html

```diff
@@ -1,30 +1,41 @@
-<nav class="navbar is-fixed-top" role="navigation" aria-label="main navigation">
-  <div class="container">
-    <div class="navbar-brand">
-      
-      <a class="navbar-item" href="{{ site.baseurl }}/">
-          <img src="{{ site.baseurl }}/images/logo.png" alt="DNAnalyzer">
+<nav class="navbar" role="navigation" aria-label="main navigation">
+  <div class="navbar-brand">
+      <a class="navbar-item logo" href="{{ site.baseurl }}/">
+          DNAnalyzer
       </a>
-
-      <a
-        role="button"
-        class="navbar-burger burger"
-        aria-label="menu"
-        aria-expanded="false"
-        data-target="top-menu"
-      >
-        <span aria-hidden="true"></span> <span aria-hidden="true"></span>
-        <span aria-hidden="true"></span>
+      <a role="button" class="navbar-burger" aria-label="menu" aria-expanded="false"
+          data-target="navbarBasicExample">
+          <span aria-hidden="true"></span>
+          <span aria-hidden="true"></span>
+          <span aria-hidden="true"></span>
       </a>
-    </div>
+  </div>
+
+  <div id="navbarBasicExample" class="navbar-menu">
+      <div class="navbar-start">
+          <a href="{{ site.baseurl }}/" class="navbar-item is-active">
+              Home
+          </a>
+
+          <a href="{{ site.baseurl }}/about" class="navbar-item">
+              About
+          </a>
+          <a class="navbar-item" href="https://streetscience.community/">
+              Street Science Community
```

Comment:
Don't know if we should also include the logo for the street science? Maybe it will break the current nice look of the website.

Reply:
I changed both the main logo and street science community logo to the original ones, which don't look bad I think. What's on your mind?